apps/foundation-app/src/app/cart/components/cart-item/cart-item.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="cart-item">
+  <img src="/assets/mh04-green_alt1.jpg">
+  <div class="cart-item__description">
+    <div class="cart-item__description__brand">Brand (HC)</div>
+    <div class="cart-item__description__name embolden">{{item.name}}</div>
```

Comment:
My reason for the "embolden" mixin over a class is that it's a lot easier to think about a "class" of an element rather than all of the classes that an element is. For me, at the very least, I think it's just a bit more concise. I don't have to go searching through all the different classes when something goes wrong. I have one class which utilizes several mixins.

Reply:
I made "embolden" into a class and a mixin like we discussed.